internal/image/image.go

```diff
@@ -7,23 +7,23 @@ import (
 	"github.com/opencontainers/go-digest"
 )
 
-// Tag is a representation of a container image Tag.
-type Tag struct {
-	Name      string
+// Image is a representation of a container image.
+type Image struct {
+	Tag       string
 	Digest    digest.Digest
 	CreatedAt *time.Time
 	semVer    *semver.Version
 }
```

Comment:
This change actually illustrates the motivation for the entire PR.

`Digest` is _not_ an attribute of a tag. It is an attribute of an _image_.